testnet/stacks-node/src/config.rs

```diff
@@ -1906,6 +1898,7 @@ impl MinerConfig {
             nonce_cache_size: 10_000,
             candidate_retry_cache_size: 10_000,
             unprocessed_block_deadline_secs: 30,
+            min_tx_count: 0,
```

Comment:
Let's do a > 0 default? It's not forcing behavior but highly encouraging it ~

Reply:
No; let's not surprise users by causing their miners to stop working altogether if this value is too high

Reply:
Broadly speaking, systemically trying to force miners to include more than X transactions in a block simply ensures that miners fill in their blocks with junk transactions by default to ensure that they can always mine (folks had the same proposition to "force" Bitcoin miners to mine non-empty blocks back in the mid-2010s, but it was never implemented for the same reason).